gammapy/visualization/tests/test_image_profile.py

```diff
@@ -0,0 +1,65 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+from numpy.testing import assert_allclose
+import astropy.units as u
+from gammapy.utils.testing import requires_data, requires_dependency, mpl_plot_check
+from astropy.coordinates import SkyCoord
+from gammapy.datasets import Datasets
+from gammapy.data import GTI
+from gammapy.estimators import MapProfileEstimator, make_orthogonal_boxes
+
+
+@requires_data()
+def make_improf():
+    datasets = Datasets.read("$GAMMAPY_DATA/fermi-3fhl-crab/",
```

Comment:
rather than using the `ProfileEstimator` you could build a very basic and small profile in a `Table` to perform a test. This avoids the complex dependency and allows more detailed testing usually.

Reply:
done